snapcraft/cli/env.py

```diff
@@ -14,7 +14,40 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import os
+import re
+from distutils import util
+from snapcraft.internal import errors
 
 
-def is_containerbuild():
-    return os.environ.get('SNAPCRAFT_CONTAINER_BUILDS')
+class ContainerConfig:
+
+    def __init__(self):
+        # The environment variable can be used in 3 ways
+        # 1. SNAPCRAFT_CONTAINER_BUILDS=1 enables local containers
```

Comment:
This is really nice, you can even make this comment block into a docstring for `__init__` with some text massaging...
"""
Defines if a container should be used and if a remote other than local should be used.
Checks for the existence of `SNAPCRAFT_CONTAINER_BUILDS` and reads its value which can be interpreted in three different ways
"""

Just an illustration.